### PR TITLE
Profile generator for seeds

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

**Comment by lennym** [comment]
It looks like this will generate random profiles as part of a normal seed run, which is something I'd like to avoid if at all possible.

No problem with having the ability to add large volumes of generated profile data, but would prefer it to be a manual opt-in. Otherwise things like regression testing become a proper pain.

**Comment by wheelsandcogs** [comment]
> It looks like this will generate random profiles as part of a normal seed run, which is something I'd like to avoid if at all possible.
> 
> No problem with having the ability to add large volumes of generated profile data, but would prefer it to be a manual opt-in. Otherwise things like regression testing become a proper pain.

Yeah, mainly included that as a demo, we can take that out.

**Comment by lennym** [comment]
Can you add the seed and test directories to `.npmignore`?

This should probably have been done some time ago, but is even more important now there's a mahoosive file in the seeds.

**Comment by wheelsandcogs** [comment]
Done.

File: lib/generate-profiles.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// eslint-disable-next-line implicit-dependencies/no-implicit
```
Comment (lennym) [inline lib/generate-profiles.js:1]: ORLY?

**Pull request merged**